scripts/release.ts

```diff
@@ -148,6 +187,135 @@ async function publishRustClient(version: string) {
 	}
 }
 
+async function getSecretFromEnvOrPrompt(envVar: string, promptMessage: string) {
```

Comment:
This can just be replaced with a simple error, no need to prompt for it. See how it's done with 1password for npm. Keep it simple.